CHANGELOG.md

```diff
@@ -19,6 +19,9 @@ Current
 
 ### Changed:
 
+- [ISSUE-17](https://github.com/yahoo/fili/issues/17) [Added pagination parameters to PreResponse] (https://github.com/yahoo/fili/pull/19) 
```

Comment:
Since this is Fili (i.e. open-source) we can't reference any internal tickets. That being said, per a request by @cdeszaq earlier (that I'm too lazy to find and link to), we shouldn't be adding Github issues after the fact. Apparently, Github issues are meant to track things todo in the future, not everything that has been done, is being done, and will be done, like Jira.